addon/components/au-modal.js

```diff
@@ -38,32 +37,9 @@ export default class AuModal extends Component {
     }
   }
 
-  @action
-  setInert(toggle) {
-    let landmarkElements = document.querySelectorAll(LANDMARKS);
-
-    this.destinationElement.inert = toggle;
-
-    landmarkElements.forEach(function (landmarkElement) {
-      if (landmarkElement.parentElement === document.body) {
-        landmarkElement.inert = !toggle;
-      }
-    });
-  }
-
   @action
   closeModal() {
-    // Prevent that the @closeModal action is run more than once.
```

Comment:
I would keep the comment until we implement #129 cause it looks very strange without an explanation 😄.